### ISSUE TITLE
scraper results hard to read - shadow background and line highlighting missing after 0.0.11.29

### ISSUE THREAD

**Issue by meesterexx** [Goldenfreddy0703/plugin.video.kaito.beta#48]
Tested this on 0.0.11.29, .31 & .36. Scraper results are had to see  now. Tested on Arctic Horizon 2 and Arctic Zephyr 2 resurrected.

See screenshots below

0.0.11.29:
![screenshot00001](https://user-images.githubusercontent.com/13154733/185683368-bd22572b-b804-4ca2-8534-6a4a4001350b.png)

0.0.11.31 & .36:
![screenshot00000](https://user-images.githubusercontent.com/13154733/185683439-11c90889-5281-442c-9291-c14c94de87ee.png)

**Comment by Goldenfreddy0703** [comment]
Oh that's not good, could you tell me which skin is used for each screenshot. I know you used Ah2 and Az2 but idk which screenshot goes to which skin.

**Comment by Goldenfreddy0703** [comment]
ok so im testing 0.0.11.36 on Ah2 and im not getting that bug where its hard to see. 
![image](https://user-images.githubusercontent.com/62456796/185708027-d6c1e62f-8102-40f3-b57f-4530cd4168fc.png)
Let me see Arctic Zephyr 2 resurrected.

**Comment by Goldenfreddy0703** [comment]
Would you mind telling me what device your using, usually linux devices can be buggy.

**Comment by meesterexx** [comment]
They are both from AH2. I first noticed it in AZ2R, so I went to check in AH2 to rule out a skin issue . When in happened in AH2, I started updating kaito in reverse (I keep several versions of the addon zip archived) until I hit .29 and the issue went away. So I think you should be able to test in either skin. 

Windows 11/Kodi 19.4. Maybe it got introduced by updating several versions of Kaito. Let me try with a fresh install.

**Comment by Goldenfreddy0703** [comment]
true, try also following my troubleshooting steps if you haven't already.

**Comment by meesterexx** [comment]
It still did it from from a clean install on AH2. So I started with a completely empty new kodi (no addons or userdata folder), only Estuary. Installed Kaito from your repo, authorized RD, put in my fanart api, set it to source select and still got the issue. I also tried it a second time with gogo and season part scraping disabled (I've had slow downs with those on in the past). Didn't make a difference.

From estuary:
![screenshot00000](https://user-images.githubusercontent.com/13154733/185711967-3a3dfd88-024b-4bd3-8be1-cf3fd5dc4ec0.png)

**Comment by meesterexx** [comment]
Downgrade to .29 immediately fixes the issue. I didn't even have to clear cache or rescrape. I haven't diffed the xml files, but there's obviously a change somewhere, the .29 screenshot doesn't have the clearlogo above the results, only the kaito raven logo, the .36 does have the clearlogo.

On Estuary:
![screenshot00001](https://user-images.githubusercontent.com/13154733/185712349-4fe30801-9d7a-435b-b2d7-395652ad5fc1.png)

**Comment by Gujal00** [comment]
@meesterexx can you go into settings and select "white" for the theme and test
![screenshot00000](https://user-images.githubusercontent.com/17394059/185712794-f6a47043-52c1-46e3-aefe-168609d571f3.png)

**Comment by Goldenfreddy0703** [comment]
Hey @Gujal00 , i just tried it with the white theme and it shows correctly for me.
![image](https://user-images.githubusercontent.com/62456796/185713206-aecc4767-2b2b-4bd2-873d-6c0e8a22ea39.png)

**Comment by meesterexx** [comment]
Lol, that was an easy fix. Thanks!!

![screenshot00002](https://user-images.githubusercontent.com/13154733/185713618-24a81085-cdb5-484f-b383-34f8b838de2b.png)

**Comment by Goldenfreddy0703** [comment]
oh, wired, i guess i should have tried my troubleshooting steps before changing the theme to white.

**Comment by Gujal00** [comment]
@meesterexx can you now switch back the theme to coloured and test again, it should continue to work. Seems to be a Kodi bug with default settings in settings.xml

**Comment by meesterexx** [comment]
Yes, it stayed fixed switching theme back from white to coloured . Nice!

**Comment by Goldenfreddy0703** [comment]
Oh ok, it was just a glitch then.

**Comment by meesterexx** [comment]
Could it have something to do with the fact that   `<setting id="general.icons">Coloured</setting>` wasn't in the settings.xml in userdata in previous versions and it's not initialized properly in the settings file until you toggle it once?

**Comment by Gujal00** [comment]
Correct, it is supposed to populate the default value mentioned in the addon settings.xml. However it is not doing it and hence the issue

**Comment by meesterexx** [comment]
My "broken" AH2 install looks like this before toggling:
`<setting id="general.icons" default="true">40389</setting>`

Then I toggle to White in the addon:
`<setting id="general.icons">White</setting>`

Then I toggle to Coloured:
`<setting id="general.icons">Coloured</setting>`

**Comment by Gujal00** [comment]
Yes, it supposed to populate the value of label 40389 from strings.po, instead it is populating the label id itself.
For e.g, this line in settings.xml
`<setting id="titlelanguage" type="labelenum" label="40369" lvalues="40370|40371" default ="40370"/>` sets the language
Similarly this line is supposed to set the theme
`<setting id="general.icons" type="labelenum" label="40388" lvalues="40389|40390" default="40389"/>`
but not happening for some reason

**Comment by meesterexx** [comment]
Ok, thanks for all the help and the explanation, much appreciated!
